react/features/filmstrip/components/native/BaseIndicator.js

```diff
@@ -0,0 +1,51 @@
+// @flow
+
+import React, { Component } from 'react';
+import { View } from 'react-native';
+
+import { Icon } from '../../../base/font-icons';
+
+import styles from './styles';
+
+type Props = {
+
+    /**
+     * True if a highlighted background has to be applied.
+     */
+    highlight: boolean,
+
+    /**
+     * The name of the icon to be used as the indicator.
+     */
+    icon: string
+};
+
+/**
+ * Implements a base indicator to be reused across all native indicators on
+ * the filmstrip.
+ */
+export default class BaseIndicator extends Component<Props> {
```

Comment:
I'd have preferred to see this done in a platform agnostic way. However that would have made this PR a lot longer and unwieldly to review. Please let's make sure we do that right after this.

Reply:
I swear I wanted to do that, but the web version uses Atlaskit and I think I'm not the one to remove that from the code. My feeling is that we should do a general cleanup and get rid of Atlaskit everywhere, but it's not my decision.